test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -521,6 +521,40 @@ def test_successful_auth_and_capture_with_normalized_stored_credential
     assert_success capture
   end
 
+  def test_successful_auth_and_capture_with_normalized_recurring_stored_credential
+    stored_credential_params = {
```

Comment:
Fun fact, there is a way to invoke standardized stored creds params in active_merchant by calling `stored_credential`. You can see the options for it in `test_helper.rb` and the `braintree_blue_test` file has some examples of how to invoke it.

Reply:
I updated all the stored credential params in this test file to use the helper 🎉 , thanks for pointing it out!